schema/v1alpha1/chassis.proto

```
@@ -0,0 +1,651 @@
+  repeated ChassisExpansionBay expansion_bays = 11;
+
+  // Chassis slots
+  repeated ChassisSlot slots = 12;
+}
+
+// TODO: Add environmental specifications to location.proto including:
+// - Operating/storage temperature ranges
+// - Operating/storage humidity ranges
+// - Maximum altitude specifications
+// Environmental specifications
+message EnvironmentalSpecs {
```

Buf (1.55.1)

symbol "schema.v1alpha1.EnvironmentalSpecs" already defined at schema/v1alpha1/location.proto:233:9

(COMPILE)